.cargo/bin/frontend.rs

```diff
@@ -0,0 +1,27 @@
+use std::{path::PathBuf, process::Command};
+
+#[cfg(windows)]
+pub const NPM: &'static str = "npm.cmd";
+
+#[cfg(not(windows))]
+pub const NPM: &'static str = "npm";
+
+pub fn main() {
```

Comment:
Is this command needed for test purposes? Because we decided that the backend should host the frontend. If it is only for test purposes, then it is better to move it to `frontend` directory. It is a little bit unexpected to see it in a shared location